third_party/qemu/README.md

```diff
@@ -1,48 +1,112 @@
-```bash
-# Run the following commands at the root of your QEMU checkout.
-touch REPO.bazel
-ln -s "/path/to/your/opentitan/repo/third_party/qemu/BUILD.qemu_opentitan.bazel" "BUILD.bazel"
-```
+QEMU's predominant advantage is its speed: internally, QEMU uses a dynamic translation backend called [TCG](https://www.qemu.org/docs/master/devel/tcg.html) (_Tiny Code Generator_) which optimizes translation of blocks of guest (e.g. RISC-V) instructions to equivalent host (e.g. x86-64) instructions to execute.
+By using QEMU, tests and custom software stacks for OpenTitan can be executed and tested by integrators, all running on a single host device without requiring any specialized hardware.
+This can often allow parallel execution at faster speeds than hardware simulation, at the cost of completely accurate behavior.
 
-Once done, every time you compile something using QEMU in OpenTitan, you need to tell Bazel to use your QEMU repository instead of downloading a release archive.
-This is done by passing the following command-line argument to bazel:
-```
---override_repository="+qemu+qemu_opentitan_src=/path/to/your/qemu/repo/"
-```
-For example:
-```bash
-./bazelisk.sh cquery --override_repository=... @qemu_opentitan//:build/qemu-system-riscv32
-```
-Since this can become quite tedious, you also have the option of adding this to your local bazelrc file. The recommended way of doing is to create a `.bazelrc-site` file at the root of the repo (if does not exists) and add:
-```
-common --override_repository=...
-```
+OpenTitan's emulated machines are currently developed on a [fork](https://github.com/lowRISC/qemu) of upstream QEMU to better facilitate the implementation of non-standard Ibex and OpenTitan features.
+There are currently two main emulation targets: Earlgrey 1.0.0, and Darjeeling (targeting the moving `master` branch).
+You can refer to the QEMU repository's documentation for more information on the development and usage of the QEMU OpenTitan machines - this page instead focuses on providing details for testing software with QEMU.
+
+## Scope
+
+A more comprehensive description of supported features and devices can be found in the relevant QEMU [Earlgrey](https://github.com/lowRISC/qemu/blob/ot-9.2.0/docs/opentitan/earlgrey.md) and [Darjeeling](https://github.com/lowRISC/qemu/blob/ot-9.2.0/docs/opentitan/darjeeling.md) documentation pages.
+Note that these may not be up-to-date compared to the implementation, and as such the source files should be treated as the root source of truth.
+Furthermore, QEMU releases are pinned in Bazel, meaning that the supported features may differ compared to when this release was taken (see the [setup guide](./setup.md)).
+
+Currently, only the Earlgrey machine (`ot-earlgrey`) has any support within OpenTitanLib for test integration, primarily on the `earlgrey_1.0.0` branch. As of the time of writing, a **non-exhaustive** list of some of the major features not emulated by QEMU include:
```

Comment:
I think there is a list of unimplemented and missing features in our QEMU docs already. Maybe we should not duplicate that list here to avoid this possibly getting out-of-sync, and point people there instead. Though I think the general notes about timing-sensitive or synchronous protocols and the like could remain here to illustrate deficiencies when compared to other testing/verification methods, up to you though.

Reply:
I think the important information that should be made upfront for someone taking a quick glance would be that some devices are not implemented at all (can give a short list of some key ones), a lot may have missing functionality and so it is advised to read the QEMU documentation for that device (where other invariants and technicalities might be documented too), and that some communication protocols are not modelled entirely right (e.g. I2C).

If you could compress the information down into general points like these, it would give a good overview of the project's state that would not require much updating besides after overhauls, but it is not the biggest issue if we have some documentation duplicated across repos.